### PR TITLE
Added render trait documentation

### PR DESCRIPTION
**Pull request by Mailaender**
I wrote some short descriptions to our various render https://github.com/OpenRA/OpenRA/wiki/Traits. Turned out what they do is not obvious to new modders.

### CODE REVIEW

**Comment by penev92** [comment]
Nice. :+1: 
Sounds like some of these traits really need to be generalized/trashed.

File: OpenRA.Mods.Common/Traits/Render/WithMakeAnimation.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Desc("Replaces the sprite during build up.")]
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Traits/Render/WithMakeAnimation.cs]: "build up" -> "construction".  The german translation for "build up" would be "anhäufen", "ansammeln", so doesn't quite fit.

**Comment by obrakmann** [comment]
Apart from that one niggle, :+1:

**Comment by Mailaender** [comment]
Fixed.

**Pull request merged**